client/components/webhook-information/index.js

```diff
@@ -2,30 +2,28 @@ import { __ } from '@wordpress/i18n';
 import { ExternalLink } from '@wordpress/components';
 import styled from '@emotion/styled';
 import interpolateComponents from 'interpolate-components';
-import { useAccount } from 'wcstripe/data/account';
 
-const WebhookEndpointText = styled.strong`
+const WebhookButtonText = styled.strong`
 	padding: 0 2px;
 	background-color: #f6f7f7; // $studio-gray-0
 `;
 
 export const WebhookInformation = () => {
-	const { data } = useAccount();
 	return (
 		<p data-testid="webhook-information">
 			{ interpolateComponents( {
 				mixedString: __(
-					"Add the following webhook endpoint {{webhookUrl/}} to your {{settingsLink}}Stripe account settings{{/settingsLink}} (if there isn't one already). This will enable you to receive notifications on the charge statuses.",
+					'Click the {{webhookUrl/}} button to {{settingsLink}}configure a webhook{{/settingsLink}}. This will complete your Stripe account connection process.',
```

Comment:
Given this isn't a URL anymore we should change this placeholder name. 

```suggestion
					'Click the {{configureButtonText/}} button to {{settingsLink}}configure a webhook{{/settingsLink}}. This will complete your Stripe account connection process.',
```